README.md

```diff
@@ -176,23 +176,120 @@ You can install the package via Composer:
 composer require spatie/laravel-permission
 ```
 
-Copy `vendor/spatie/laravel-permission/config/permission.php` to `config/permission.php`
+Copy the required files:
 
-Copy `vendor/spatie/laravel-permission/database/migrations/create_permission_tables.php.stub` to `database/migrations/2017_12_01_010101_create_permission_tables.php`
+```bash
+cp vendor/spatie/laravel-permission/config/permission.php config/permission.php
+cp vendor/spatie/laravel-permission/database/migrations/create_permission_tables.php.stub database/migrations/2018_01_01_000000_create_permission_tables.php
+```
 
-In `bootstrap/app.php`, add the following code below other service providers:
+You will also need to create another configuration file at `config/auth.php`. You should paste the content below:
 
 ```php
-$app->register(Spatie\Permission\PermissionServiceProvider::class);
-$app->configure('permission');
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Defaults
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default authentication "guard" and password
+    | reset options for your application. You may change these defaults
+    | as required, but they're a perfect start for most applications.
+    |
+    */
+
+    'defaults' => [
+        'guard' => env('AUTH_GUARD', 'api'),
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Guards
+    |--------------------------------------------------------------------------
+    |
+    | Next, you may define every authentication guard for your application.
+    | Of course, a great default configuration has been defined for you
+    | here which uses session storage and the Eloquent user provider.
+    |
+    | All authentication drivers have a user provider. This defines how the
+    | users are actually retrieved out of your database or other storage
+    | mechanisms used by this application to persist your user's data.
+    |
+    | Supported: "token"
+    |
```

Comment:
`Supported: "token"` vs the `driver=>'api'` below.

Is there an update needed here for consistency?

Reply:
Well, I'm not sure. If you take a look at the original file, you'll see the same thing: https://github.com/laravel/lumen-framework/blob/5.5/config/auth.php Maybe I'm missing something, so that's why I kept it.

Reply:
I can accept that :)